_data/sites.json

```
@@ -1808,6 +1808,20 @@
         ]
     },
 
+    {
+        "name": "BeMyEye",
+        "url": "https://bemyeye.zendesk.com/hc/en-us/articles/360009596613-How-do-I-delete-my-account-",
+        "difficulty": "limited",
```

```suggestion
        "difficulty": "hard",
```

I think this should be `hard`, we only use `limited` for those where the deletion is only possible if covered by a GDPR policy or similar.

---

The account will be deleted only if you have not withdrawn any money, otherwise will be deactivated and the data will remain.